_docs/services/cloud-gov-identity-provider.md

```diff
@@ -23,24 +23,42 @@ Plan Name | Description |
 To create a service instance that can provision identity provider credentials, run the following command:
 
 ```sh
-cf create-service cloud-gov-identity-provider oauth-client my-uaa-client
+cf create-service cloud-gov-identity-provider oauth-client <SERVICE_INSTANCE_NAME>
```

Comment:
i was trying to make it more clear here that `my-uaa-client` was something user-provided and not part of the command.